src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -1832,7 +1834,8 @@ private bool ConsumeSingleLineComment(ReadOnlySpan<byte> localBuffer, int previo
                 return false;
             }
 
-            ValueSpan = _buffer.Slice(previousConsumed, idx + 2);   // Include the double slash and potential line feed at the end of the comment as part of it.
+            // Exclude // at the beginning of the comment
+            ValueSpan = _buffer.Slice(previousConsumed + 2, idx);
```

Comment:
Is `idx` set to the length that will point `ValueSpan` to right before the newline character(s) or do we need `idx - 1` or `idx -2` here?

Reply:
`idx` is set right before the line separator.